client/layout/404.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDom from 'react-dom';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+import { setSection } from 'state/ui/actions';
+
+let Page404 = React.createClass( {
+	displayName: 'Page404',
+
+	statics: {
```

Comment:
Wouldn't it better to move this method away from this component? It isn't the only possible way to use this component. In my opinion we should rather create local function closer to where it's really used.